fix(css): font-stretch has been renamed to font-width (#35267)
mdn/content: files/en-us/web/css/font-stretch/index.md

{{CSSRef}}

> [!NOTE]

> > The `font-stretch` property [has now been renamed to `font-width`](https://drafts.csswg.org/css-fonts/#font-stretch-desc) in the specifications. And the `font-stretch` name remains as an alias for the `font-width` property.

Reviewer: ```suggestion
> The `font-stretch` property [has now been renamed to `font-width`](https://drafts.csswg.org/css-fonts/#font-stretch-desc) in the specifications. The name `font-stretch` has been kept as an alias for the `font-width` property.
```

> > The new name `font-width` has not been supported by any browser yet.

Reviewer: ```suggestion
> The new name `font-width` is not yet supported by any browsers.
```

## Formal syntax

{{csssyntax("font-width")}}

## Examples

### Setting font stretch percentages

```html
<p class="condensed">an elephantine lizard</p>
<p class="normal">an elephantine lizard</p>
<p class="expanded">an elephantine lizard</p>
```

```css
@font-face {
  src: url("fonts/LeagueMonoVariable.ttf");
  font-family: "LeagueMonoVariable";
  font-style: normal;
  font-stretch: 1% 500%; /* Required by Chrome */
}

p {
  font:
    1.5rem "LeagueMonoVariable",
    sans-serif;
}

.condensed {
  font-stretch: 50%;
}

.normal {
  font-stretch: 100%;
}

.expanded {
  font-stretch: 200%;
}
```

> {{EmbedLiveSample("Setting font stretch percentages", "100%", 200)}}

Reviewer: Good idea, but this doesn't seem to be working in the preview. Do you want to have the font locally, as in https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-alternates#enabling_swash_glyphs, or are you supposed to load it from mdn/shared-assets?

Author: Yes, it's there in the shared-assets.

> [!NOTE]
> The `font-stretch` property was initially defined in CSS 2, but dropped in CSS 2.1 due to the lack of browser implementation. It was brought back in CSS 3.

> > The property [has now been renamed to `font-width`](https://drafts.csswg.org/css-fonts/#font-stretch-desc) in the specifications.

Reviewer: Suggestions:
- put this note at the top of the page, and explicitly say that `font-stretch` is now defined in the specifications as an alias for `font-width`, which is identical in behavior, but that the `font-width` name is not yet supported by any browsers.
- remove the pre-existing note from here
- also pass `font-width` to csssyntax, which should fix the formal syntax error.
